packages/schema-blocks/src/functions/BlockHelper.ts

```diff
@@ -34,4 +34,15 @@ function restoreBlock( clientId: string, removedBlock: string, removedAttributes
 	dispatch( "core/block-editor" ).replaceBlock( clientId, block );
 }
 
-export { getBlockByClientId, removeBlock, restoreBlock };
+/**
+ * Finds the block type in the list of registered blocks in the block editor.
+ *
+ * @param {string} blockName The block to search for.
+ *
+ * @return {BlockInstruction} The found block.
+ */
+function getBlockType( blockName: string ) {
```

Comment:
I am missing a TypeScript return type:
```suggestion
function getBlockType( blockName: string ): Block | undefined {
```

Reply:
Do not forget to import the type definition of `Block` at the top:
```js
import { Block, BlockInstance, createBlock } from "@wordpress/blocks";
```